crates/context_aware_config/src/api/context/handlers.rs

```diff
@@ -55,6 +60,8 @@ pub fn endpoints() -> Scope {
         .service(delete_context)
         .service(bulk_operations)
         .service(list_contexts)
+        .service(move_handler)
+        .service(delete_context)
```

Comment:
Why add these routes again? Remove these

Reply:
it got added while merging conflicts will remove this